Octokit/Http/SimpleJsonSerializer.cs

```diff
@@ -92,6 +92,17 @@ public override object DeserializeObject(object value, Type type)
                 {
                     if (ReflectionUtils.GetTypeInfo(type).IsEnum)
                     {
+                        //first try to get all custom attributes
```

Comment:
Maybe we should hardcoded strings. Because i have no idea how we can deserialize enum properly without to loop trough all the enum values and check if it has an attribute or not.